content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -133,6 +142,42 @@ This requires you have `openssl` installed on your machine.
     /tmp/pod-cert-chain-workload.pem: OK
     {{< /text >}}
 
+## (Optional) Generating root and intermediate keys and certificates offline
+
+This section gives you instructions on how to generate the keys and certificates offline and apply to
+the Istio cluster.
+
+On an offline machine with good security (restricted access), follow the `Create the root pair` and the
+`Create the intermediate pair` sections in this [guide](https://jamielinux.com/docs/openssl-certificate-authority/)
+to create the keys and certificates for your root and intermediate CA.
+
+After the certificates and keys are created,
+the root certificate is in `/root/ca/certs/ca.cert.pem`,
+the intermediate certificate is in `/root/ca/intermediate/certs/intermediate.cert.pem` and the intermediate key
+is in `/root/ca/intermediate/private/intermediate.key.pem`. We need to bundle them and copy them into the machine
+to manage the cluster.
+
+{{< text bash >}}
+$ mkdir ca-keycerts
+$ cp /root/ca/certs/ca.cert.pem ca-keycerts/root-cert.pem
+$ cp /root/ca/intermediate/certs/intermediate.cert.pem ca-keycerts/ca-cert.pem
+$ cp /root/ca/intermediate/private/intermediate.key.pem ca-keycerts/ca-key.pem
+$ cp ca-keycerts/ca-cert.pem ca-keycerts/cert-chain.pem
+$ tar cvf ca-keycerts.tar ca-keycerts
+{{< /text >}}
+
+Use a portable data storage device to copy `ca-keycerts.tar` to the machine configured to manage to the Istio cluster.
+On the machine, create the secret `cacerts`:
+
+{{< text bash >}}
+$ tar xvf ca-keycerts.tar
+$ kubectl create secret generic cacerts -n istio-system --from-file=ca-keycerts/ca-cert.pem \
+    --from-file=ca-keycerts/ca-key.pem --from-file=ca-keycerts/root-cert.pem --from-file=ca-keycerts/cert-chain.pem
+{{< /text >}}
+
+After that, follow the
```

Comment:
Delete "After that"

`1. [Plug in the existing certificate and key] created in step (whatever step the number is).